packages/amplify-cli/src/domain/amplify-usageData/UsageDataPayload.ts

```diff
@@ -13,7 +12,7 @@ export class UsageDataPayload implements IUsageDataPayload {
   installationUuid: string;
   amplifyCliVersion: string;
   input: CommandLineInput | null;
-  inputOptions: InputOptions;
+  inputOptions: Pick<CommandLineInput, 'options'>;
```

Comment:
Just confirming you don't want `CommandLineInput['options']` instead? ie this object still has everything nested under "options"

Reply:
You're absolutely right, great catch.